custom_components/teslafi/sensor.py

```diff
@@ -246,6 +246,26 @@
         available=lambda u, d, h: u and d.tpms.rear_right,
     ),
     # endregion
+    # region TeslaFi API Counts
+        TeslaFiSensorEntityDescription(
+        key="commands",
+        name="API Commands",
+        icon="mdi:counter",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=True,
```

Comment:
I'm not sure yet if these need to be enabled by default. I would probably set either hidden by default, or disabled by default. 

The reason is because for many/most people, they won't know what this is or what it means, since it's not related to their car. So in that case I think it would just be confusing if it appeared with little explanation. For people that do want or need it, it's easy to go in and enable the entities. That's also in line with how a lot of the other entities here work: things that are not likely to be very commonly used, get configured as disabled by default so as not to clutter the device or dashboards (this wouldn't get added to a dashboard automatically at least, because of the diagnostic category)